src/plugins/displayLayout/components/DisplayLayout.vue

```diff
@@ -226,6 +226,11 @@ export default {
     this.composition.load();
     this.gridDimensions = [this.$el.offsetWidth, this.$el.scrollHeight];
 
+    // Initialize objectStyles if it doesn't exist for older versions
+    if (!this.domainObject.configuration.objectStyles) {
+      this.domainObject.configuration.objectStyles = {};
```

Comment:
See comment above. We don't mutate object models until the user changes something.

Reply:
This still needs to be addressed. The interceptor should be handling this now